### PR TITLE
adding change 1

### PR DESCRIPTION
**Pull request by Eng-raghad**
Change 1 is added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-
+adding new change 1
```
Comment (Eng-raghad) [inline README.md:1]: Requesting changes.

Comment (raghadalfattal) [inline README.md:1]: comment 1

**Review by raghadalfattal — CHANGES_REQUESTED** [review]
Requesting change

File: README.md (hunk 1 above)
Comment (Eng-raghad) [reply README.md:1]: Change applied.

**Review by raghadalfattal — APPROVED** [review]

**Comment by raghadalfattal** [comment]
LGTM

**Pull request merged**